packages/block-library/src/rss/block.json

```diff
@@ -46,6 +46,22 @@
 		"html": false,
 		"interactivity": {
 			"clientNavigation": true
+		},
+		"__experimentalBorder": {
+			"radius": true,
+			"color": true,
+			"width": true,
+			"style": true,
+			"__experimentalDefaultControls": {
+				"radius": true,
+				"color": true,
+				"width": true,
+				"style": true
+			}
```

Comment:
I'm not sure all the new controls should be exposed by default.

Reply:
`This would be good to confirm again with our design and product experts though.
`
For now, just removed the default controls.

Reply:
Looks like only the border controls were prevented from displaying by default. Padding and margin controls need to be explicitly made optional.

```json
"__experimentalDefaultControls": {
	"padding": false,
	"margin": false
}
```